tokio-postgres/src/bind.rs

```diff
@@ -20,8 +20,9 @@ where
     I: IntoIterator<Item = P>,
     I::IntoIter: ExactSizeIterator,
 {
+    type BytesResult = Result<bytes::Bytes, Error>;
```

Comment:
What is this change for?

Reply:
After the `impl From` block was introduced, the compiler required the type hint. Replacing the `impl From` block will make this change unnecessary.